### ISSUE TITLE
Filter activites by a username pattern

### ISSUE THREAD

**Issue by kmoppel** [zalando/pg_view#47]
Currently there's a "--pid" flag but in practice pid's can change so fast that tracking by username would be more helpful

**Comment by alexeyklyukin** [comment]
yeah, thanks. Also it would be useful to have tracking by some regexp in the query (or username). Additionally, one should be able to show the related queries even if they are idle (right now even pid-related tracking do not show the idle queries for this pid).

**Comment by LappleApple** [comment]
@alexeyklyukin Do you want to add a "Help Wanted" label?
